plugins/inputs/modbus/README.md

```diff
@@ -20,6 +20,14 @@ The Modbus plugin collects Discrete Inputs, Coils, Input Registers and Holding R
  ## Timeout for each request
  timeout = "1s"
 
+ ## Maximum number of retries and the time to wait between retries
+ ## when a slave-device is busy.
+ ## NOTE: Please make sure that the overall retry time (#retries * wait time)
+ ##       is always smaller than the query interval as otherwise you will get
+ ##       an "did not complete within its interval" warning.
```

Comment:
Let's remove this note, I think this isn't the right advice.  It may actually be desired to configure the plugin to exceed the interval.  This way you are more likely to get a response, and not get into a situation where the device never fully reply in time.

Reply:
Hmm, this note was requested earlier by @ssoroka:
>Agh. we might not be exposing that setting to the plugin. May not be possible.. Ok maybe just a note in the settings not to pick a value that would end up retrying longer than your interval.

Could you guy please resolve this conflict and tell me if I should leave that comment in or not... ;-)

Reply:
We can resolve this post merge.